keras_cv/utils/bounding_box.py

```diff
@@ -149,3 +149,33 @@ def pad_bounding_box_batch_to_shape(bounding_boxes, target_shape, padding_values
     return tf.pad(
         bounding_boxes, paddings, mode="CONSTANT", constant_values=padding_values
     )
+
+
+def mask_to_bbox(masks):
```

Comment:
lets try to suppose batch dimension before we submit this!  Thank you!

Reply:
hmm, let me think how to apply mapping along a axis 
i.e of apply_along_axis in Numpy.